Library/BookCopy.aspx

```diff
@@ -0,0 +1,67 @@
+﻿<%@ Page Language="C#" AutoEventWireup="true" CodeBehind="BookCopy.aspx.cs" Inherits="Library.BookCopy" %>
+<%@ import namespace="System.Data" %>
+
+<!DOCTYPE html>
+
+<html xmlns="http://www.w3.org/1999/xhtml">
+<head runat="server">
+    <title></title>
+</head>
+<body>
+    <form id="form1" runat="server">
+        <h1>Book Directory</h1>
```

Comment:
This is a bit of a nitpick, but I'd keep my page names and page headings consistently named.